### PR TITLE
Add Tweets User Mentions Api

### PR DESCRIPTION
**Pull request by wangmeng255**
I am not sure where should call getUserMentionsTweets in ClientApp, just put it in onMapInit method and use test id 1350123171933003776. Please review. Thanks

### CODE REVIEW

**Comment by davidmjiang** [comment]
> I am not sure where should call getUserMentionsTweets in ClientApp, just put it in onMapInit method and use test id 1350123171933003776. Please review. Thanks

Hey meng, we won't need to call this from clientApp, it will be called someone on the server. However, have you got this to work? I tried calling the api from postman and got this:
image

File: src/controller.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+const twitterBearToken = require('../config').twitter.bearToken;
```
Comment (davidmjiang) [inline src/controller.js:1]: typo. bearerToken

Comment (davidmjiang) [reply src/controller.js:1]: image

**Comment by wangmeng255** [comment]
Hi @davidmjiang I make it like http://localhost:3000/api/usermentionstweets?twitterId=1350123171933003776. you can have a try. I will remove changes in clientapp

File: src/controller.js (hunk 1 above)
Comment (wangmeng255) [reply src/controller.js]: Fixed it. Sorry bearer, hello bear

**Comment by davidmjiang** [comment]
Ok, the api is working now. I wonder if we should be cleaning up the response so that we get back only what we're interested in, which is the array of tweets. Or do you think that will be done at a different step? 
image

**Review by davidmjiang — APPROVED** [review]

**Pull request merged**